docs/reference/indices/get-field-mapping.asciidoc

```diff
@@ -48,7 +48,11 @@ include::{docdir}/rest-api/common-parms.asciidoc[tag=index-ignore-unavailable]
 (Optional, boolean) If `true`, the response includes default mapping values.
 Defaults to `false`.
 
-include::{docdir}/rest-api/common-parms.asciidoc[tag=local]
+`local`::
+deprecated:[7.8.0, Field mappings are always retrieved locally]
+(Optional, boolean) If `true`, the request retrieves information from the local
+node only. Defaults to `false`, which means information is retrieved from
+the master node.
```

Comment:
I would add that the parameter currently is just a noop.

Can you also update the [migrate_7_8.asciidoc file](https://github.com/elastic/elasticsearch/blob/7.x/docs/reference/migration/migrate_7_8.asciidoc) when backporting this change to the 7.x branch?

Reply:
Added.  Thanks for the heads up about the migrate doc. Will do for 7.x.